tests/Integration/Framework/DocBlockResolverAware/DocBlockResolverCustomServicesAwareTest.php

```diff
@@ -15,14 +16,16 @@ final class DocBlockResolverCustomServicesAwareTest extends TestCase
     public static function setUpBeforeClass(): void
     {
         DirectoryUtil::removeDir(__DIR__ . '/.gacela');
-        CustomServicesPhpCache::resetCache();
     }
 
     protected function setUp(): void
     {
         Gacela::bootstrap(__DIR__, static function (GacelaConfig $config): void {
             $config->resetInMemoryCache();
             $config->addAppConfig('config/custom-services/*.php');
+            $config->registerGenericListener(static function (GacelaEventInterface $event): void {
+                // dump('Triggered -> ' . \get_class($event)); # useful for debugging
+            });
```

Comment:
This helped me a lot debugging the behaviour by rendering all events that were triggered. Especially the new ones related to the "class name cache" scope.